src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_help.py

```diff
@@ -261,3 +261,139 @@
+        - name: --condition -c
+          short-summary: Add condition. Accept space separated conditions in 'FIELD=VALUE' format. The possible values
+                         for the field are (case-insensitive) 'resourceId', 'category', 'caller', 'level',
+                         'operationName', 'resourceGroup', 'resourceProvider', 'status', 'subStatus', 'resourceType',
+                         or anything beginning with 'properties.'. If a condition with the same field exists, its value
+                         will be updated.
+        - name: --action -a
+          short-summary: Add action group. Accept space separated resource id of the action groups to be added. The
+                         invalid resource id is ignored.
+        - name: --webhook-properties -w
+          short-summary: Space separated web hook properties in 'key[=value]' format. These properties will be
+                         associated with the action groups added in this command. For any webhook receiver in these
+                         action group, these data are appended to the webhook payload. To attach different webhook
+                         properties to different action groups, add the action groups in separate update-action
+                         commands.
+"""
+
+helps['monitor activity-log alert enable'] = """
+    type: command
+    short-summary: Enable this activity log alert
+"""
+
+helps['monitor activity-log alert disable'] = """
+    type: command
+    short-summary: Disable this activity log alert
+"""
+
+helps['monitor activity-log alert update-condition'] = """
+    type: command
+    short-summary: Update the conditions of this activity log alert.
+    parameters:
+        - name: --name -n
+          short-summary: Name of the activity log alerts
+        - name: --add-condition -a
+          short-summary: Add condition. Accept space separated conditions in 'FIELD=VALUE' format. The possible values
+                         for the field are (case-insensitive) 'resourceId', 'category', 'caller', 'level',
+                         'operationName', 'resourceGroup', 'resourceProvider', 'status', 'subStatus', 'resourceType',
+                         or anything beginning with 'properties.'. If a condition with the same field exists, its value
+                         will be updated.
+        - name: --remove-condition -r
+          short-summary: Remove conditions. Accept space separated conditions in 'FIELD=VALUE' format. The removing is
+                         executed before adding. Some condition, such as 'category', is required for this alert to be
+                         operational, therefore remove without replace with a new one will cause service error.
+        - name: --reset
+          short-summary: Remove all the existing conditions before add new conditions. When --reset is given, the values
+                         of --remove-condition will be ignored.
```

Comment:
Alternatively, you could simply accept `--remove-condition *` to remove all conditions.  Intuitive and minimizes arguments.